README.md

```diff
@@ -877,31 +877,86 @@ interface FIDAttribution {
 ```ts
 interface INPAttribution {
   /**
-   * A selector identifying the element that the user interacted with for
-   * the event corresponding to INP. This element will be the `target` of the
-   * `event` dispatched.
+   * A selector identifying the element that the user first interacted with
+   * as part of the frame where the INP candidate interaction occurred.
+   * If `interactionTarget` is an empty string, that generally means the
+   * element was removed from the DOM as part of the interaction.
```

Comment:
I'm not sure this is 100% accurate (though maybe "generally" gives us enough leeway).

We've seen instances where the empty string is returned. Plus we're planning on recording this even if it is removed from the DOM (what's the latest on that @mmocny ?)

Reply:
I think we should stick to my current wording. I don't think we need to document all scenarios that could lead to an empty string. The main thing I want to make sure is clear is that the string CAN be empty (e.g. this value won't always have a selector).